swap_meet/item.py

```diff
@@ -1,2 +1,11 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return ("Hello World!")
+    
+    def condition_description(self):
+        condition_statements = ["Terrible","Bad","OK","Good","Excellent"]
+        return condition_statements[self.condition-1]
```

Comment:
Per our discussion in class, you can use `round` or `int` for this method to work for float values of condition. Otherwise, nice appraoch!